src/pages/FormPage.scss

```diff
@@ -17,4 +21,37 @@ tds-toast {
     position: fixed;
     right: var(--tds-spacing-element-32);
     bottom: var(--tds-spacing-element-32);
-}
\ No newline at end of file
+}
+
+.input-wrapper {
+    padding: 32px;
+    background: var(--tds-grey-50);
+    border-radius: 4px;
+    transition: filter 250ms ease;
+}
+
+.spinner-container {
+    position: absolute;
+    bottom: -100px;
+    left: 50%;
+    transform: translateX(-50%);
+    opacity: 0;
+    transition: bottom 250ms ease,  opacity 250ms ease;
+}
+
+.form-sending {
+    .input-wrapper {
+        filter: blur(4px);
+    }
+    .spinner-container {
+        opacity: 1;
+        bottom: 100px;
+    }
+}
+
+#anonymous-toggle {
```

Comment:
Why use an id?

Reply:
I wanted to target only one specific item.
However, I can change it to a more general name and make it a class so it can be used multiple times.

Reply:
Made it a utility class instead.